src/python/pants/engine/legacy/source_mapper.py

```diff
@@ -64,6 +64,7 @@ def _iter_owned_files_from_hydrated_target(self, legacy_target):
         yield f
 
     # Handle `resources`-declaring targets.
+    # TODO: Remember to get rid of this in 1.5.0.dev0, when the deprecation of `resources` is complete.
```

Comment:
See my next change... I wouldn't add a conditional here though, the ones on python_target are enough. This is just a reminder to kill this code...